torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -44,6 +44,14 @@ inline Tensor prelu(const Tensor& input, const Tensor& weight) {
   return torch::prelu(input, weight);
 }
 
+inline Tensor relu(Tensor& input, const ReLUOptions& options) {
```

Comment:
I think `options` can be optional in Python version, and it would be great to have a test for `relu(input)` case as well :)